server/middleware/setUpSentry.ts

```diff
@@ -4,13 +4,7 @@ import type express from 'express'
 import { gitRef } from '../applicationVersion'
 import config from '../config'
 
-function setUpSentryErrorHandler(app: express.Express): void {
-  if (config.sentry.dsn) {
-    app.use(Sentry.Handlers.errorHandler() as express.ErrorRequestHandler)
```

Comment:
As per https://docs.sentry.io/platforms/javascript/guides/express/migration/v7-to-v8/#updated-sdk-initialization point 4, individual handlers were removed.